.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+[//]: # (This is a comment: NOTE, you only need to keep the follwoing format if you are filing a bug report)
```

Comment:
Typo (follwoing)

Also, I can't find any documentation about this comment syntax? In any case I'll point out that someone who doesn't recognize a comment won't know that means it shouldn't show up in the output. ;)